client/jetpack-connect/help-button.jsx

```diff
@@ -1,35 +1,40 @@
 /** @format */
+
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import Gridicon from 'gridicons';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
 import { localize } from 'i18n-calypso';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+export class JetpackConnectHelpButton extends PureComponent {
+	static propTypes = { label: PropTypes.string };
 
-const JetpackConnectHelpButton = ( { label, translate, onClick } ) => {
-	return (
-		<LoggedOutFormLinkItem
-			className="jetpack-connect__help-button"
-			href="https://jetpack.com/contact-support"
-			target="_blank"
-			rel="noopener noreferrer"
-			onClick={ onClick }
-		>
-			<Gridicon icon="help-outline" size={ 18 } />{' '}
-			{ label || translate( 'Get help setting up Jetpack' ) }
-		</LoggedOutFormLinkItem>
-	);
-};
+	recordClick = () => void this.props.recordTracksEvent( 'calypso_jpc_help_link_click' );
```

Comment:
Any benefit to have the `void` here? It doesn't seem anything is using the returned value.

Reply:
Probably not, but this way I'm _sure_ the returned value is not used.

When using these single-expression type arrow functions as event handlers, I always consider what the implicit return could mean.

Back in the day, returning `false` from event handlers had weird effects. That's probably meaningless now, but this in the sense that `recordTracksEvent` is a side-effect with an irrelevant return, I think the `void` expresses that nicely.